src/avram/upsert.cr

```diff
@@ -100,5 +123,9 @@ module Avram::Upsert
         \{% raise "Please use the 'upsert_lookup_columns' macro in #{@type} before using '{{ method.id }}'" %}
```

Comment:
![image](https://user-images.githubusercontent.com/6721355/151651405-927cdf30-32e8-48c6-9caf-028d52422248.png)
This is an interesting case
I was passing a map directly, which didn't fall through to self.upsert(Array), OR the standard upsert, but was caught up by this method's error when I passed an Enumerable (from the map)